stolon/templates/keeper-ro-service.yaml

```diff
@@ -16,5 +16,11 @@ spec:
     - port: {{ .Values.ports.externalPort }}
       targetPort: {{ .Values.ports.internalPort }}
   selector:
+    # Due to https://github.com/sorintlab/stolon/pull/433/commits/38ae6b13b5e161a5bfe0fbe01084ca060eaf2e76#diff-95cdd374e9440fde010ff35d65f8cf3fR54
+    {{- if eq .Values.store.backend "kubernetes" }}
+    app: "stolon-keeper"
```

Comment:
I did deploy two clusters in same namespace for K8. why'd you think this does not work? 
Anyway, this could be a good approach to remove cluttered if statements.

I introduced them in the first place in order to not change existing behaviour.

Reply:
Without suggested rework of  labels, services in both clusters will endup balancing traffic between all keepers (If I don't miss anything):

```
  selector:
    app: stolon-keeper
    component: keeper
```

https://github.com/lwolf/stolon-chart/blob/master/stolon/templates/keeper-ro-service.yaml#L18-L20